tideways.c

```diff
@@ -2551,6 +2552,8 @@ void hp_init_trace_callbacks(TSRMLS_D)
     cb = tw_trace_callback_view_class;
     register_trace_callback("Mage_Core_Block_Abstract::toHtml", cb);
     register_trace_callback("Magento\\Framework\\View\\Element\\AbstractBlock::toHtml", cb);
+    register_trace_callback("Neos\\Flow\\Mvc\\View\\JsonView::render", cb);
+    register_trace_callback("Neos\\FluidAdaptor\\View\\AbstractTemplateView::render", cb);
```

Comment:
I don't understand this comment, if its not needed we don't need to add it. Or do we want to add the Fluid function call itself?